portfolio/src/main/webapp/Javascript/main.js

```diff
@@ -4,6 +4,20 @@ $("#my-form").submit(function (e) {
     url: "/form-handler",
     type: "post",
     data: $("#my-form").serialize(),
+  }).done(function(){
+    document.getElementById("txt").value = "";
```

Comment:
I think you can write this with jQuery as:
```suggestion
    $("#txt").value = "";
```